Lib/idlelib/editor.py

```diff
@@ -678,8 +678,13 @@ def goto_line_event(self, event):
         if lineno <= 0:
             text.bell()
             return "break"
-        text.mark_set("insert", "%d.0" % lineno)
+        newindex = f'{lineno}.0'
+        text.mark_set("insert", newindex )
```

Comment:
Rogue extra space after `newindex`.

Reply:
In response to next comment, I reverted mark_set change.